CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.4.2 (2021-09-21)
+
+This release contains bug fixes since the 2.4.1 release.  We deem it
+high priority to upgrade.
+
+**Bugfixes**
```

Comment:
Are these all the bug fixes we have ready right now, or is that taking into account the ones we wrap up on Monday like we discussed?

Reply:
Those were only merged at that point initially.